Makefile

```diff
@@ -3,7 +3,7 @@ clean:
 	./scripts/clean.sh
 
 .PHONY: ensure
-ensure: clean
```

Comment:
Removing this because it's not necessary in CI, and there are times when you want to ensure but _not_ clean.